client/templates/initiatives/initiatives.js

```diff
@@ -7,9 +7,9 @@ Template.initiatives.helpers({
       }), cursor);
```

Comment:
Ahh good point.. It feels not very dry.. I am typing same code in several places... wonder how we can dry it out... Do something like Initiatives.active().find() would collection helpers be able to help?